src/PixelFlags.cc

```diff
@@ -28,11 +28,13 @@
 #include "lsst/afw/table/Source.h"
 #include "lsst/meas/base/PixelFlags.h"
 
+#include <cctype> // ::toupper
+#include <algorithm> // std::transform
+
 namespace lsst { namespace meas { namespace base {
-namespace {
 
 template <typename MaskedImageT>
-class FootprintBits : public afw::detection::FootprintFunctor<MaskedImageT> {
+class PixelFlagsAlgorithm::FootprintBits : public afw::detection::FootprintFunctor<MaskedImageT> {
```

Comment:
Moved back to anonymous namespace now that the updateflags function was also moved to the anonymous namespace